website/cue/reference/remap/functions/parse_regex_all.cue

```diff
@@ -34,7 +34,7 @@ remap: functions: parse_regex_all: {
 		},
 	]
 	internal_failure_reasons: [
-		"`value` fails to parse using the provided `pattern`.",
+		"`value` is not a string.",
```

Comment:
Thanks @mdeltito. I think this is implied for each function argument and we probably don't want to repeat it across functions. Also, the (current) wording is also confusing since the argument name is `value` but we also refer to VRL `value` resolution. IMO we can just leave `internal_failure_reasons` empty.

Reply:
fair enough, updated!